infra/aws/terraform/artifacts.k8s.io/main.tf

```
@@ -0,0 +1,63 @@
+/*
+Copyright 2022 The Kubernetes Authors.
```

```suggestion
Copyright The Kubernetes Authors.
```
Year is old. Not sure if year from copyright notices are allowed here. It saves a lot of hassle.